src/UserSpaceInstrumentation/OrbitUserSpaceInstrumentation.cpp

```diff
@@ -169,10 +169,10 @@ bool& GetIsInPayload() {
   }
   is_in_payload = true;
 
-  thread_local const pid_t tid = orbit_base::GetCurrentThreadIdNative();
+  thread_local const pid_t kTid = orbit_base::GetCurrentThreadIdNative();
```

Comment:
This is `thread_local`. I think kCamelCase is fine but I wanted to point out how this is quite a special case.